### ISSUE TITLE
Adapt design to accommodate new fee model

### ISSUE THREAD

**Issue by taleldayekh** [beamer-bridge/beamer#618]
## Specifications

- [ ] Sync with @weilbith, @manuelwedler for details on specifications.

For which platform and in what size/aspect ratio?

**Comment by weilbith** [comment]
@olympe-silve correct me if I'm wrong, but if I'm not mistaken this is already done as we worked on the overall design.

**Comment by olympe-silve** [comment]
@weilbith that's right, hence it is in my "Done" section in zenhub. I close the issue.
